data.sql

```diff
@@ -4,3 +4,8 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight) VAL
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight) VALUES ('Gabumon', '2018-11-15', 2, true, 8.0);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight) VALUES ('Pikachu', '2021-01-07', 1, false, 15.04);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight) VALUES ('Devimon', '2017-05-12', 5, true, 11.0);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight) VALUES ('Charmander', '2020-02-08', 0, false, 11.0);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight) VALUES ('Plantmon', '2021-11-15', 2, true, 5.7);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight) VALUES ('Squirtle', '1993-04-02', 3, false, 12.13);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight) VALUES ('Angemon', '2005-06-12', 0, true, 45.0);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight) VALUES ('Boarmon', '2005-06-07', 7, true, 20.4);
```

Comment:
- [ ] Kindly check the screenshot below and insert the data as per project requirements. For example I cannot find the animal with name Ditto inserted in your database. Looking also at the screenshot you will notice some weights are negative values.
![Screen Shot 2023-02-21 at 7 36 10 PM](https://user-images.githubusercontent.com/62849872/220515419-e724d6e8-b2ee-45b2-82bc-c0319a7604a0.png)

Reply:
Done 🚀